packages/opds-web-client/src/components/context/__tests__/StoreContextTest.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
```

Comment:
The convention we are using for tests is nameOfComponent-text.tsx instead of nameOfComponentTest.tsx